observability/cockpit/concepts.mdx

```diff
@@ -0,0 +1,174 @@
+
+</Concept>
+
+<Concept>
+
+## Alerting rules
+
+Alerting rules allow you to define criteria that determine whether an alert will fire. The rule consists of queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met. When an alert fires, a notification is sent to the alert manager, which forwards the notification to receivers.
+
+</Concept>
+
+<Concept opened>
+
+## Cockpit
+
+A Cockpit is an Instance of the Observability product that has a dedicated Grafana instance. A Scaleway Project can have only one Cockpit.
+
+</Concept>
+
+<Concept>
+
+## Contact points
+
+Contact points define how your contacts are notified when an alert fires. They have contact point types such as emails, slack, texts etc. When an alert fires, a notification is sent to all contact point types listed for a contact point.
+
+</Concept>
+
+<Concept>
+
+## Endpoints
+
+An endpoint is the point of entry in a communication channel when two systems are interacting. The endpoint is the means from which the API can access the resources they need from a server to perform their task. An endpoint can include a URL of a server or service.
+
+</Concept>
+
+<Concept>
+
+## Loki Remote Write
+
+Loki Remote Write is a feature that allows you to send metrics from almost any device to a Loki server.
+
+</Concept>
+
+<Concept>
+
+## LogQL
+
+LogQL is Grafana Loki’s language for querying logs. LogQL uses labels and operators for filtering.
+
+</Concept>
+
+<Concept>
+
+## Logs
+
+Logs provide a record of all events and errors that take place during the lifecycle of your resources. They represent an excellent source of visibility if you want to know when a problem occurred, or which events correlate with it.
+
+</Concept>
+
+<Concept>
+
+## Managed alerts
+
+Managed alerts are alerts criteria that Scaleway defines for you. They allow you to receive alerts for behaviors that we deem abnormal on your products.
+
+</Concept>
+
+<Concept>
+
+## Managed dashboards
+
+A managed dashboard is a set of one or more panels organized into one or more rows where you can visualize your data.
+
+</Concept>
+
+<Concept>
+
+## Metric
+
+A metric is a lifecycle-related numerical representation of data (e.g.disk usage and CPU usage) measured over intervals of time. Metrics give you a bird's eye view of your infrastructure.
+
+</Concept>
+
+<Concept>
+
+## Monitoring
+
+Monitoring is a tool that allows you to watch and understand the state of your systems. Monitoring is based on gathering predefined sets of metrics or logs.
+
+</Concept>
+
+<Concept>
+
+## Prometheus Remote Write
+
+Prometheus remote write is a feature that allows you to send your metrics from almost any device to a Prometheus server.
```

Comment:
I think either this should be in title case, or the name of the concept shouldn't